azext_iot/monitor/handlers/common_handler.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+import re
+import yaml
+
+from azext_iot.monitor.base_classes import AbstractBaseEventsHandler
+from azext_iot.monitor.parsers.common_parser import CommonParser
+
+
+class CommonHandler(AbstractBaseEventsHandler):
+    """
+    Handles messages as they are read from egress event hub.
+    Use this handler if you aren't sure which handler is right for you.
+
+    Keyword Args:
+        device_id       (str)   only process messages sent by this device
+        devices         (list)  only process messages sent by these devices
+        pnp_context     (bool)  interpret the device as being a pnp device
+        interface_name  (str)   expected interface name of pnp device
+        content_type    (str)   assumed content type (utf-8, ascii, etc)
+        properties      (list)  list of properties to extract from message headers
+        output          (str)   output format (json, yaml, etc)
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
I'm not crazy about this kwargs style. Normally kwargs are used for flags, options and less commonly used arguments.